tests/http_functions/trigger/main.py

```diff
@@ -0,0 +1,3 @@
+def main(req):
+    print("AAAAAAAAAAAAA", flush=True)
```

Comment:
Did you intend to commit this in this PR?

Reply:
Good catch